www/include/Administration/parameters/general/form.php

```diff
@@ -32,7 +33,11 @@
  * For more information : contact@centreon.com
  *
  */
-require_once _CENTREON_PATH_ . "www/class/centreonGMT.class.php";
+
+require_once __DIR__ . '/../../../../../bootstrap.php';
+require_once __DIR__ . "/../../../../class/centreonGMT.class.php";
+
+use Symfony\Component\Dotenv\Dotenv;
```

Comment:
if you use Dotenv you can retrieve your file as follow:

$dotenv = new Dotenv();
$dotenv->load(__DIR__.'/.env');

then access your .env variables as $_ENV['MY_ENV_VARIABLE_NAME']

https://symfony.com/doc/4.1/components/dotenv.html

Reply:
This line should be removed
I think it's better to get parameter from container, cause if the value is not defined in environment, it uses the default value set in the config